cmd/gnofaucet/gnofaucet.go

```diff
@@ -73,6 +87,7 @@ type serveOptions struct {
 	Memo               string `flag:"memo" help:"any descriptive text"`
 	TestTo             string `flag:"test-to" help:"test addr (optional)"`
 	Send               string `flag:"send" help:"send coins"`
+	Captcha            bool	  `flag:"captcha" help:"enable or disable catpcha(defalut: false)"`
```

Comment:
```suggestion
	CaptchaSecret      string	  `flag:"captcha" help:"recaptcha secret (if empty, captcha are disabled)"`
```

You should replace the Boolean check and the hardcoded secret with a customizable secret, and check for `captchaEnabled := CaptchaSecret != ""`